subsystems/DiscordRPC/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    java
+    `java-library`
+}
+
+apply(from = "$rootDir/config/gradle/publish.gradle")
```

Comment:
What does this need from `publish.gradle`?

do we expect subsystems to be published as independent artifacts to the jar repository?

Reply:
I think publishing is possible in future.

Really i can use common. Because i needs repositories from it

Reply:
Even though it might be possible in the future we should probably not include it right now. Using `common` might indeed be a better option.

Reply:
Ah, okay. Maybe still something to review later, but `common` doesn't worry me as much as `publish`. 👍